docs/modes-of-operation.rst

```diff
@@ -254,6 +254,12 @@ Additionally, if a secondary selects multiple autoprimaries for a zone based on
 Adding a autoprimary can be done either directly in the database,
 or by using the 'pdnsutil add-autoprimary' command.
 
+.. warning::
+  When a secondary receives notification while bootstraping a new domain using autosecondary feature, it will send
+  SOA and NS queries to the IP address obtained from the ``supermasters`` table. These queries are **not** recursive.
```

Comment:
```suggestion
  When a secondary receives notification while bootstrapping a new domain using autosecondary feature, it will send
  SOA and NS queries to the IP address matched in the ``supermasters`` table. These queries are **not** recursive.
```